### PR TITLE
refactor: team settings redesign

### LINKED ISSUE #11999: [CAL-2627] Refactor: Team Settings Pages
null

<sub>[CAL-2627](https://linear.app/calcom/issue/CAL-2627/refactor-team-settings-pages)</sub>

### PR DESCRIPTION
**Pull request by Udit-takkar**
Fixes: https://github.com/calcom/cal.com/issues/11999

Appearance Page

Screenshot 2023-11-04 at 1 29 11 AM

Members Page

Screenshot 2023-11-04 at 1 34 01 AM

Team Profile Page

(ignore team_deletion_cannot_be_undone. It is there in en.json)

Screenshot 2023-11-04 at 1 34 33 AM

### CODE REVIEW

**Review by sean-brydon — DISMISSED** [review]
This setting redesign is looking bliss! 

LGTM - tested on preview

File: packages/features/ee/components/BrandColorsForm.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                    } catch (err) {
+                      setDarkModeError(true);
+                    }
+                  }}
+                />
+                {darkModeError ? (
+                  <div className="mt-4">
+                    <Alert
+                      severity="warning"
+                      message="Dark Theme color doesn't pass contrast check. We recommend you change this colour so your buttons will be more visible."
```
Comment (sean-brydon) [inline packages/features/ee/components/BrandColorsForm.tsx:115]: NIT: we could use i18n in the future

**Comment by PeerRich** [comment]
NIT: spacing between disable cal.com branding and hide "book a team member" button is higher than above
CleanShot 2023-11-15 at 09 57 23@2x

**Review by sean-brydon — DISMISSED** [review]
LGTM - thanks for fixing i18n and spacing

**Review by PeerRich — APPROVED** [review]

**Pull request merged**